src/js/components/DataTable/DataTable.js

```diff
@@ -122,6 +123,8 @@ const DataTable = ({
     select,
   ]);
 
+  const [childExpand, setChildExpand] = useState([]);
```

Comment:
Upon reflection, I think we call this `rowExpand`, because it's the "row" we are expanding, not the "child" we are expanding.

Reply:
Addressed the change